app/_how-tos/build-custom-docker-image.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Build a custom Docker image
+description: "How to build a custom Docker image "
+
+products:
+    - gateway
+
+content_type: how_to
+
+tldr: 
+  q: How do I build a custom {{site.base_gateway}} Docker image?
+  a: Create a Docker file and use [`docker build`](/how-to/run-docker-images/) to build the image.
+
+works_on:
+    - on-prem
+
```

Comment:
```suggestion
faqs:
  - q: What can I do with a {{site.base_gateway}} custom image?
     a: You can use a custom image to specify certain settings, like custom [ports](/gateway/network-ports-firewall/) or [`kong.conf`](/gateway/manage-kong-conf/) parameters. This can be useful if your organization has certain requirements or other software that they use in conjunction with their API gateway. You can also use custom images in automation pipelines.
```
Double check for accuracy, but I wanted to capture what we talked about for use cases here.